src/patternfly/components/Login/login.scss

```
@@ -150,119 +171,107 @@
-  align-items: center;
-  padding-top: var(--pf-c-login__main-footer-link--PaddingTop);
-  padding-bottom: var(--pf-c-login__main-footer-link--PaddingBottom);
-  color: var(--pf-c-login__main-footer-link--Color);
-
-  @media (min-width: $pf-global--breakpoint--md) {
-    grid-gap: var(--pf-c-login__main-footer-link--GridGap);
-  }
+.pf-c-login__main-footer-links {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: var(--pf-c-login__main-footer-links--PaddingTop) var(--pf-c-login__main-footer-links--PaddingRight) var(--pf-c-login__main-footer-links--PaddingBottom) var(--pf-c-login__main-footer-links--PaddingLeft);
+  margin-bottom: var(--pf-c-login__main-footer-links--MarginBottom);
 
   &:hover,
   &.pf-m-hover {
     --pf-c-login__main-footer-link--Color: var(--pf-c-login__main-footer-link--hover--Color);
   }
 }
 
-.pf-c-login__main-footer-link-text {
-  // styles the text to be hidden from the screen but read by a screen reader.
-  @media (max-width: $pf-global--breakpoint--md) {
-    position: fixed;
-    overflow: hidden;
-    clip: rect(0, 0, 0, 0);
-    white-space: nowrap;
-    border: 0;
-  }
+.pf-c-login__main-footer-links-item {
+  padding-right: var(--pf-c-login__main-footer-links-item--PaddingRight);
+  padding-left: var(--pf-c-login__main-footer-links-item--PaddingLeft);
+  margin-bottom: var(--pf-c-login__main-footer-links-item--MarginBottom);
 }
 
-.pf-c-login__footer {
-  grid-area: footer;
+.pf-c-login__main-footer-links-item-link {
+  svg {
+    fill: var(--pf-c-login__main-footer-links-item-link-svg--Fill);
+    width: var(--pf-c-login__main-footer-links-item-link-svg--Width);
+    height: var(--pf-c-login__main-footer-links-item-link-svg--Height);
+  }
 
-  @media (max-width: $pf-global--breakpoint--sm) {
-    padding-right: var(--pf-c-login__footer--sm--PaddingRight);
-    padding-left: var(--pf-c-login__footer--sm--PaddingLeft);
+  &:hover {
+    svg {
+      fill: var(--pf-c-login__main-footer-links-item-link-svg--hover--Fill);
+    }
   }
 }
 
-.pf-c-login__main-footer-grid {
-  display: grid;
-  grid-gap: var(--pf-c-login__main-footer-grid--GridGap);
-  grid-template-columns: var(--pf-c-login__main-footer-grid--GridTemplateColumns);
+.pf-c-login__main-footer-band {
+  padding: var(--pf-c-login__main-footer-band--PaddingTop) var(--pf-c-login__main-footer-band--PaddingRight) var(--pf-c-login__main-footer-band--PaddingBottom) var(--pf-c-login__main-footer-band--PaddingLeft);
+  color: var(--pf-c-login__main-footer-band--Color);
+  text-align: center;
+  background-color: var(--pf-c-login__main-footer-band--BackgroundColor);
+}
 
-  @media (max-width: $pf-global--breakpoint--md) {
-    justify-items: center;
-  }
-
-  @media (min-width: $pf-global--breakpoint--md) {
-    .pf-c-login__main-footer-grid-item {
-      .pf-c-button {
-        padding-left: 0;
-      }
-    }
+.pf-c-login__main-footer-band-item {
+  &:first-child {
+    padding-bottom: var(--pf-c-login__main-footer-band-item);
```

```suggestion
    padding-bottom: var(--pf-c-login__main-footer-band-item--PaddingBottom);
```

However, I might re-write this selector like this to allow for more lines in the future without a CSS change to create this gap.

```
.pf-c-login__main-footer-band-item {
  > * + * {
    padding-top: var(--pf-c-login__main-footer-band-item--PaddingTop);
  }
}
```